app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```

Comment:
I18n cho toàn bộ text trong tất cả các file html chú ah

dùng `link_to` thay vì tag `a` nhé